Source/Scene/Cesium3DTileset.js

```diff
@@ -786,6 +802,68 @@ define([
 
     ///////////////////////////////////////////////////////////////////////////
 
+    var scratchJulianDate = new JulianDate();
+
+    function updateExpireDate(tile) {
```

Comment:
This - and a lot of the date management should be moved out of this file - and into the `Cesium3DTile` class.